fusestore/store/src/api/rpc/flight_service_test.rs

```diff
@@ -361,3 +363,100 @@ async fn test_scan_partition() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+#[test(tokio::test)]
+async fn test_user_apis() -> anyhow::Result<()> {
+    use common_flights::StoreClient;
+
+    let addr = crate::tests::start_store_server().await?;
+
+    let mut client = StoreClient::try_create(addr.as_str(), "root", "xxx").await?;
+
+    let name = "test_user";
+    let pass = "test_pass";
+    let salt = "test_salt";
+    let pass_hash: [u8; 32] = Sha256::digest(pass.as_bytes()).into();
+    let salt_hash: [u8; 32] = Sha256::digest(salt.as_bytes()).into();
+
+    // new user
+    client.add_user(name, pass, salt).await?;
+
+    // duplicated username
+    let r = client.add_user(name, pass, salt).await;
+
+    assert!(r.is_err());
+    assert_eq!(
+        r.unwrap_err().code(),
+        ErrorCode::UserAlreadyExists("").code()
+    );
+
+    // get user
+    let user = client.get_user_info(name).await?;
+    assert!(user.user_info.is_some());
+    let u = user.user_info.unwrap();
+    assert_eq!(u.name, name);
+    assert_eq!(u.password_sha256, pass_hash,);
+    assert_eq!(u.salt_sha256, salt_hash,);
+
+    // get users
+    let users = client.get_users_info(&vec![name]).await?;
```

Comment:
What about using `list_user_info` to retrieve multiple user info records,  while `get_user_info` to retrieve a single user info record.

Reply:
There is a `get_user_info` method as you described. 

Maybe this ut case is misleading though (should have passed multiple names to `get_users_info`).

~~~
  pub fn get_user_info( &self, username: impl AsRef<str>) -> Result<GetUserInfoActionResult>

  pub fn get_users_info<T: AsRef<str>>(&self, usernames: &[T]) -> Result<GetUsersInfoActionResult>
    
  pub fn get_all_users_info(&self) -> Result<GetAllUsersInfoActionResult>

~~~

Reply:
name is ok to me.

Remove `info`?
```
get_user
get_users
get_all_users
```